src/App.js

```diff
@@ -17,9 +22,27 @@ function App() {
         >
           Learn React
         </a>
+        <div>{props.dummyText}</div>
+        <div>{props.defaultText}</div>
+        <ul>
+          {books.map((book) => {
+            return (
+            <li>{book.title}</li>
```

Comment:
- [ ] I got a console error here:
`Warning: Each child in a list should have a unique "key" prop.`

Let's add unique key identifier to each book.